components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -636,6 +642,11 @@ func NewProvisioningProcessingQueue(ctx context.Context, provisionManager *provi
 			stage: createRuntimeStageName,
 			step:  provisioning.NewBusolaMigratorOverridesStep(),
 		},
+		{
+			stage:    createRuntimeStageName,
```

Comment:
after the merg of the [PR](https://github.com/kyma-project/control-plane/pull/912) we need to add a condition which will enable the step only for kyma 2.0

Reply:
We would also like to install agent for central monitoring for existing 1.X SKRs in stage so that we can cross verify the new monitoring functionality with the existing one.